### PR TITLE
[Wheel] Added configuration for openvono-dev

### PR DESCRIPTION
**Pull request by mryzhov**
This PR added configuration for openvino-dev wheel package and will be used in our CI only:
1) Added setup.cfg for runtime package to include EULA and 3rd-party licenses.
2) Added openvino-dev.setup.cfg for dev package (will be used in CI to build the wheel package)
3) Added common requirements - openvino-dev.requirements.txt

### CODE REVIEW

**Review by AlexKoff88 — COMMENTED** [review]
Requirements look good to me from the POT standpoint. I guess we need to align and fix versions (`==`) for all the components. With this change, we can guarantee the results on validated platforms.

File: inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+defusedxml>=0.5.0
+scipy==1.2.1
+jstyleson==0.0.2
+numpy==1.16.3
+addict==2.2.1
+pandas==0.24.2
+hyperopt==0.1.2
+networkx==2.2
+tqdm==4.31.1
+py-cpuinfo==4.0.0
```
Comment (eaidova) [inline inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt:10]: does it planed to be used with DL Workbench?
If I right remember, they have py-cpuinfo==7.0.0 as requirements

File: inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+pillow>=2.6.1
+scikit-image
+scikit-learn
+yamlloader
+shapely
+nibabel
+pydicom
+sentencepiece
+tokenizers
+editdistance
```
Comment (eaidova) [inline inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt:21]: which principies has this requirements list? Should it includes all requirements for the tools or not?
I am not sure that all AC requirements here (missed nltk, parasail, fast-ctc-decode, rawpy)

File: inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+jstyleson==0.0.2
+numpy==1.16.3
+addict==2.2.1
+pandas==0.24.2
+hyperopt==0.1.2
+networkx==2.2
+tqdm==4.31.1
+py-cpuinfo==4.0.0
+PyYAML
+pillow>=2.6.1
```
Comment (eaidova) [inline inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt:12]: sometime ago, vulnerabilities was detected in Pillow, recommended to update on 8.1 and higher

File: inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt (hunk 2 above)
Comment (mryzhov) [reply inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt:21]: I've taken those requirements from your setup.py. It should include all mandatory requiremens  for the tool
If we missed something, please define them here

File: inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt (hunk 3 above)
Comment (eaidova) [inline inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt:12]:
```suggestion
pillow>=8.1.0
```

File: inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt (hunk 2 above)
Comment (eaidova) [inline inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt:21]:
```suggestion
editdistance
parasail
fast-ctc-decode
rawpy
nltk
```

File: inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt [hunk 4]
```diff
@@ -0,0 +1,22 @@
+defusedxml>=0.5.0
+scipy==1.2.1
```
Comment (eaidova) [inline inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt:2]: @AlexKoff88 @MaximProshin could you please suggest another scipy version (it does not support python3.8+ and can not be installed on windows with such python version)

Comment (AlexKoff88) [reply inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt:2]: @eaidova, let's put it this way. What version you need for AC. We could adjust POT: update to the recent version of AC and verify that it works.

Comment (eaidova) [reply inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt:2]: we do not pin version explicitly, but currently in testing used 1.5.4, is it ok for you?

Comment (AlexKoff88) [reply inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt:2]: we need to verify that.

File: inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt (hunk 1 above)
Comment (AlexKoff88) [reply inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt:10]: This actually comes from AC, POT does not use it directly as far as I can see). So again, the task is to gather all the requirements across OV components, find misalignments, find the common denominator for each dependency (probably the highest version), fix versions, and let teams validate them.

Comment (eaidova) [reply inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt:10]: in AC this component has py-cpuinfo!=5.0,!=6.0.0 (these 2 version broken for working in frozen environment)
My question more about, is there is plans to migrate DL Workbench docker on pypi dev package usage or not. For us current version is ok, there is no conflict between components inside dev-package.

Not sure that we need the whole requirements alignment across all OV components in wheel creation scope, this task is wider than represented tools.

File: inference-engine/ie_bridges/python/wheel/setup.py [hunk 5]
```diff
@@ -302,6 +306,9 @@ def get_package_dir(install_cfg):
 if os.path.exists(package_license):
     copyfile(package_license, "LICENSE")
 
+packages = find_packages(','.join(get_dir_list(PY_INSTALL_CFG)))
+package_data = {}
```
Comment (ilyachur) [inline inference-engine/ie_bridges/python/wheel/setup.py:310]: Is it ok that previously `package_data` was equal to result of `find_data_files(get_dir_list(LIB_INSTALL_CFG))` and currently it is always empty dictionary?

Comment (mryzhov) [reply inference-engine/ie_bridges/python/wheel/setup.py:310]: dev package doesn't  include ie_libs and can be empty, but for runtime package package_data will be added in generate_package()

File: inference-engine/ie_bridges/python/src/openvino/inference_engine/__init__.py [hunk 6]
```diff
@@ -7,8 +7,8 @@
     #
     # If you're using a custom installation of openvino,
     # add the location of openvino dlls to your system PATH.
-    openvino_dll = os.path.join(os.path.dirname(__file__), "..", "..", "..", "openvino.libs")
-    os.environ["PATH"] = os.path.abspath(openvino_dll) + ";" + os.environ["PATH"]
+    openvino_dlls = os.path.join(os.path.dirname(__file__), "..", "..", "openvino", "libs")
```
[the remark this answers is not in the document]
Comment (mryzhov) [reply inference-engine/ie_bridges/python/src/openvino/inference_engine/__init__.py]: removed from this PR

File: inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt (hunk 1 above)
Comment (mryzhov) [reply inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt]: updated to py-cpuinfo!=5.0,!=6.0

File: inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt (hunk 4 above)
Comment (mryzhov) [reply inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt]: Updated: scipy==1.5.4

File: inference-engine/ie_bridges/python/wheel/setup.py (hunk 5 above)
Comment (mryzhov) [reply inference-engine/ie_bridges/python/wheel/setup.py]: excluded from this PR

File: inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt (hunk 1 above)
Comment (AlexKoff88) [reply inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt]: We just added "texttable==1.6.3" in POT. POT also requires a torch cpu version. It is not installed by default but it is requested to be installed if the user is going to use specific optimization methods.

**Review by eaidova — APPROVED** [review]

**Review by generalova-kate — APPROVED** [review]

File: inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt (hunk 1 above)
Comment (mryzhov) [reply inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt]: Added texttable==1.6.3, but how do you suggest handling the torch cpu version?

**Review by AlexKoff88 — APPROVED** [review]

File: inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt (hunk 1 above)
Comment (AlexKoff88) [reply inference-engine/ie_bridges/python/wheel/meta/openvino-dev.requirements.txt]: As per our discussion, let keep torch out of the sope for now.

**Pull request merged**